tcmu-runner/build/build_rpm

```diff
@@ -9,6 +9,8 @@ if test "$DISTRO" != "fedora" -a "$DISTRO" != "centos" -a "$DISTRO" != "rhel"; t
     exit 0
 fi
 
+RELEASE="$(lsb_release --short -r | cut -d '.' -f 1)" # sytem release
```

Comment:
you might get into trouble here, since we use Xenial for all Debian-based builds like Jessie, or Trusty. Was this variable somehow not defined from the parent job?

Reply:
Copied-and-pasted from samba's build_rpm (which is not applicable for Debian-based systems).